nodeup/pkg/model/ntp.go

```diff
@@ -35,6 +35,13 @@ type NTPBuilder struct {
 
 var _ fi.ModelBuilder = &NTPBuilder{}
 
+type NtpDaemon string
```

Comment:
Perhaps it doesn't have to be public?

Reply:
sure I have unexport the const.  Shall I do the same with the func too?

Reply:
Oh right, that would be good.